src/sys/stat.rs

```diff
@@ -50,6 +50,19 @@ pub fn mknod<P: ?Sized + NixPath>(path: &P, kind: SFlag, perm: Mode, dev: dev_t)
     Errno::result(res).map(drop)
 }
 
+/// Create a special or ordinary file
+/// ([see mknodat(2)](http://man7.org/linux/man-pages/man2/mknodat.2.html)).
```

Comment:
Better to reference POSIX docs than Linux docs
http://pubs.opengroup.org/onlinepubs/9699919799/functions/mknod.html

Reply:
The POSIX one is not super helpful for endusers. It does not even mention errnos. I would rather include no manpage link, then link to the posix one.

Reply:
nix is a portable crate, not a Linux crate.  The Linux manpage is only useful for Linux users.  But the POSIX man page can be relied upon by all nix users.

Reply:
But POSIX deviate from reality in subtle ways (hence the use of `shall`). Nix is not fully portable anyway, since it only provides a thin layer above systems libc unlike projects like gnulib. So you have to actually look at every manpage of every operating system you care about to be sure, it won't break.

Reply:
This would be actually be the more honest documentation: https://www.gnu.org/software/gnulib/manual/gnulib.html#mknodat

Reply:
That's irrelevant, because nix doesn't bind to gnulib; it binds to the system's C library.  The POSIX documentation is the GCD of all supported platforms.  While some platforms may deviate, the POSIX docs are still the single most relevant source.